src/hypervisor.adoc

```diff
@@ -1583,7 +1583,7 @@ are documented in <<tinst-vals>>.
 .Machine trap instruction (`mtinst`) register.
 include::images/bytefield/mtinstreg.edn[]
 
-`mtinst` is a *WARL* register that need only be able to hold the values that
+`mtinst` is a *WARL* register that only needs to be able to hold the values that
```

Comment:
@wmat @AFOliveira this and the above were not typos; it is syntactically correct English.  The new text is fine, so you don't need to revert the PR, but going forward, please don't treat this grammatical pattern as a bug.

Reply:
Pondering it a bit, I think it reads better the old way around, so I'm going to push a revert of these two.  (The one down below is obviously a correct fix; thanks for that.)